### DISCUSSION TITLE
Network Request Error, Received Status 400

### DISCUSSION THREAD

**Discussion by BakungaBronson** [reach-sh/reach-lang#1240] [Q&A] [answered]
Hello,

So when I do a Reach Run, the program compiles successfully and starts. I then enter the first input but on the second input stage when I am asked for an address, it takes the address and returns an error.

![image](https://user-images.githubusercontent.com/51344005/177726880-b366bcaf-a0b5-460e-88c8-b4c9ca13e145.png)

Any suggestions on how I can fix this?

**Comment by jeapostrophe** [comment, marked as answer]
There error says exactly what the problem is. You are trying to set a map for an address that has not interacted with the contract, which is not possible on Algorand. https://docs.reach.sh/networks/#p_9
